src/charts/iot-edge-connector/values.yaml

```diff
@@ -7,14 +7,23 @@ edgeproviderimage:
   secretKey: hub0-cs
 vkimage:
   repository: microsoft/virtual-kubelet
-  tag: 0.3
+  tag: latest
   pullPolicy: Always
 env:
   nodeName: iot-edge-connector-hub0
-  nodeTaint:
+  nodeOsType: Linux
+  monitoredNamespace: ""
+  # 'provider' must be 'web'
+  provider: web
+taint:
+  enabled: true
+  key: virtual-kubelet.io/provider
+  value: iotedge
```

Comment:
Should we also update the toleration in `sample-deployment.yaml` to reflect the values given here?

Reply:
Yes, will do that today and open up another PR for the sample deployment files and also for the README file.

Reply:
:+1: